landoapi/revisions.py

```diff
@@ -132,6 +134,32 @@ def check_author_planned_changes(*, revision, **kwargs):
     return "The author has indicated they are planning changes to this revision."
 
 
+def check_relman_approval(relman_phid, supported_repos):
+    """Check that Release Managers group approved a revision on a repo
+    with approval required"""
+
+    def _check(*, revision, repo, **kwargs):
+
+        # Check if this repository needs an approval from relman
+        local_repo = supported_repos.get(repo["fields"]["shortName"])
+        assert local_repo is not None, "Unsupported repository"
+        if local_repo.approval_required is False:
+            return None
+
+        # Check that relman approval was requested and that the
+        # approval was granted
+        reviewers = get_collated_reviewers(revision)
+        relman_review = reviewers.get(relman_phid)
+        if relman_review is None:
+            return "The release-managers group was not requested for review"
+        if relman_review["status"] == ReviewerStatus.ACCEPTED:
+            return None
+
+        return "The release-managers group did not accept that stack: you need to wait for a group approval from release-managers, or request a new review."  # noqa
```

Comment:
Can you switch this to using implied continuation inside paranthesis? We prefer that to the long lines and `# noqa` for things that aren't urls etc. ex:
```Python
return (
    "The release-managers group did not accept that stack: you "
    "need to wait for a group approval from release-managers, or "
    "request a new review."
)
```